docker/configure.sh

```diff
@@ -1,21 +1,14 @@
 #!/bin/bash
 
 export JETBOT_VERSION=0.4.2
+echo "export JETBOT_VERSION=$JETBOT_VERSION"
 
 L4T_VERSION_STRING=$(head -n 1 /etc/nv_tegra_release)
 L4T_RELEASE=$(echo $L4T_VERSION_STRING | cut -f 2 -d ' ' | grep -Po '(?<=R)[^;]+')
 L4T_REVISION=$(echo $L4T_VERSION_STRING | cut -f 2 -d ',' | grep -Po '(?<=REVISION: )[^;]+')
 
 export L4T_VERSION="$L4T_RELEASE.$L4T_REVISION"
-
-if [[ "$L4T_VERSION" == "32.4.3" ]]
-then
-    # docker hub
-    export JETBOT_DOCKER_REMOTE=jetbot
-elif [[ "$L4T_VERSION" == "32.4.4" ]]
-then
-    export JETBOT_DOCKER_REMOTE=jetbot
-fi
+echo "export L4T_VERSION=$L4T_RELEASE.$L4T_REVISION"
```

Comment:
Instead of deleting JETBOT_DOCKER_REMOTE.  I would recommend hard-setting setting a default value of

export JETBOT_DOCKER_REMOTE=jetbot

This is the smallest change to accomplish what I believe the intent was (to handle L4T versions which are not specified here).

Reply:
ok